example/src/main/java/io/github/btkelly/gandalf/example/activities/SplashActivity.java

```diff
@@ -49,6 +49,13 @@ public int contentView() {
     //** Methods below are just for allowing switching of bootstrap file, not related to Gandalf implementation **//
 
     @Override
+    protected void onStop() {
+        super.onStop();
+        MockWebServerUtil.setMockBootstrapRes(this, R.raw.no_action_bootstrap);
+        System.exit(0);
```

Comment:
This is to fix the issue where you are stuck with the force update call as you are always blocked when trying to get into the app. This allows you to see each scenario once and on restart it will revert back to no action where you can again select which one to simulate.